cluster/gce/manifests/etcd.manifest

```diff
@@ -9,14 +9,17 @@
 "securityContext": {
     "seccompProfile": {
         "type": "RuntimeDefault"
-    }
+    },
+    "runAsUser": {{runAsUser}},
```

Comment:
nit: looking at the other instances of values templated in the manifest this should be `{{ run_as_user }}`

Reply:
fixed